docs/user/library/jdbc/oracle.rst

```diff
@@ -80,7 +80,15 @@ Advanced
 |     ``timeout``       | a session with the database listener           |
 |                       | (milliseconds)                                 |
 +-----------------------+------------------------------------------------+
+|``Get column remarks`` | Boolean flag specifies whether column REMARKS  |
```

Comment:
@turingtestfail could you make another PR to rename this flag as "Get remarks" so that it can be later reused for table remarks?